cmd/kubectl-gadget/deploy.go

```diff
@@ -258,6 +260,12 @@ func init() {
 	deployCmd.PersistentFlags().BoolVar(
 		&disallowGadgetsPull,
 		"disallow-gadgets-pulling", false, "Disallow pulling gadgets from registries")
+	deployCmd.PersistentFlags().BoolVar(
```

Comment:
Thanks for working on this!

Could you please update [exporting metrics documentation](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/docs/reference/export-metrics.mdx) and also [profile blockio documentation](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/gadgets/profile_blockio/README.mdx#exporting-metrics)?